splinepy/helpme/create.py

```diff
@@ -298,6 +299,188 @@ def revolved(
     return type(spline)(**spline_dict)
 
 
+def swept(
+    cross_section,
+    trajectory,
+    nsections=None,
+    cross_section_normal=None,
+):
+    """Sweeps a cross-section along a trajectory
+
+    Parameters
+    ----------
+    crossection : Spline
+      Cross-section to be swept
+    trajectory : Spline
+      Trajectory along which the cross-section is swept
+    nsections : int
```

Comment:
As discussed, we should accept locations

Reply:
I've added an auto_refinement-routine which refines the trajectory at the points of highest curvature. Auto-Refinement can be handled via a Boolean in the function call of `splinepy.helmpe.create.swept(traj, cs, cs_nv, auto_refinement)`.

Reply:
I removed auto_refinement and hand over the responsibility to the user in terms of a fine enough trajectory.